### PR TITLE
C4-35 Scrub bcrypt/cryptacular

### PR DESCRIPTION
**Pull request by willronchetti**
- We have to upgrade our EB platforms to Python latest (3.6)
- There is a bug in cryptacular such that it cannot be compiled on EB
- We don't use it anyway (offloaded to Auth0) so it has to be scrubbed.

### CODE REVIEW

File: snovault/storage.py [hunk 1]
```diff
@@ -868,12 +868,8 @@ class Blob(Base):
     data = Column(types.LargeBinary)
 
 
-# User specific stuff
-import cryptacular.bcrypt
-crypt = cryptacular.bcrypt.BCRYPTPasswordManager()
-
 def hash_password(password):
-    return crypt.encode(password)
+    return ''  # we do not use this
```
Comment (netsettler) [inline snovault/storage.py]: This seems dangerous to leave in place. Is there some reason we don't either remove this function or make it raise an error if called?

Comment (willronchetti) [reply snovault/storage.py]: Fixed.

**Comment by willronchetti** [comment]
New changes look good as well, feel free to merge

**Comment by willronchetti** [comment]
No longer needed, Kent has a PR that does this

**Pull request closed without merging**